packages/server/src/async/producers/notifications/measurable-state-changed.js

```diff
@@ -6,16 +6,38 @@ const { MeasurableState } = require('./measurable-state');
 
 class MeasurableStateChanged extends MeasurableState {
 
-  constructor(options) {
-    super(options);
+  /**
+   * @param {Models.Measurable} measurable
+   */
+  constructor(measurable) {
+    super(measurable);
     this.templateName = Producer.TEMPLATE_NAME.MEASURABLE_STATE_IS_CHANGED;
   }
 
+  /**
+   * @protected
+   * @return {Promise<boolean>}
+   */
+  async _isActual() {
```

Comment:
Can you describe what this means, perhaps with a better name? ``isActual`` doesn't mean much to me.

Reply:
When triggers like this "MeasurableStateChanged" are fired first of all they should detect "does an event correspond to them?".